site/src/pages/TemplatePage/DAUChart.tsx

```diff
@@ -0,0 +1,123 @@
+import useTheme from "@material-ui/styles/useTheme"
+
+import { Theme } from "@material-ui/core/styles"
+import {
+  BarElement,
+  CategoryScale,
+  Chart as ChartJS,
+  ChartOptions,
+  defaults,
+  Legend,
+  LinearScale,
+  LineElement,
+  PointElement,
+  Title,
+  Tooltip,
+} from "chart.js"
+import { Stack } from "components/Stack/Stack"
+import { HelpTooltip, HelpTooltipText, HelpTooltipTitle } from "components/Tooltips/HelpTooltip"
+import { WorkspaceSection } from "components/WorkspaceSection/WorkspaceSection"
+import dayjs from "dayjs"
+import { FC } from "react"
+import { Line } from "react-chartjs-2"
+import * as TypesGen from "../../api/typesGenerated"
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  BarElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+)
+
+export interface DAUChartProps {
+  templateDAUs: TypesGen.TemplateDAUsResponse
+}
+export const Language = {
+  loadingText: "DAU stats are loading. Check back later.",
+  chartTitle: "Daily Active Users",
+}
```

Comment:
This is the old pattern. As mentioned before, the new pattern is to create a json file in [here](https://github.com/coder/coder/tree/main/site/src/i18n/en) and then use the `t` function [like so](https://github.com/coder/coder/blob/3cc888f9710234f17337de073f846721fd8326a4/site/src/pages/TemplatePage/TemplatePage.tsx#L27).

Not a huge deal but in the future we should shoot for consistency 👍🏼